modules/alpha/moduletemplate-keda-v0.0.1-alpha.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: operator.kyma-project.io/v1alpha1
+kind: ModuleTemplate
+metadata:
+  name: moduletemplate-keda
+  namespace: kcp-system
+  labels:
+    "operator.kyma-project.io/managed-by": "lifecycle-manager"
+    "operator.kyma-project.io/controller-name": "manifest"
+    "operator.kyma-project.io/module-name": "keda"
+  annotations:
+    "operator.kyma-project.io/module-version": "0.0.1-alpha"
+    "operator.kyma-project.io/module-provider": "internal"
+    "operator.kyma-project.io/descriptor-schema-version": "v2"
+spec:
+  target: remote
+  channel: alpha
+  data:
+    apiVersion: operator.kyma-project.io/v1alpha1
+    kind: Keda
+    metadata:
+      labels:
+        operator.kyma-project.io/kyma-name: keda-sample
+      name: keda-sample
+    spec:
+      logging:
+        operator:
+          level: "debug"
+  descriptor:
+    component:
+      componentReferences: []
+      name: kyma.project.io/module/keda
+      provider: internal
+      repositoryContexts:
+      - baseUrl: europe-west4-docker.pkg.dev/sap-se-cx-kyma-wookiee/dev-registry/unsigned
+        componentNameMapping: urlPath
+        type: ociRegistry
+      resources:
+      - access:
+          digest: sha256:e231a9b5dd60142699b323e9921bcad986aca29036af2f0b6ffe8a1747eba5e6
+          type: localOciBlob
+        name: keda
+        relation: local
+        type: helm-chart
+        version: 0.0.1-alpha
+      - access:
+          digest: sha256:f4a599c4310b0fe9133b67b72d9b15ee96b52a1872132528c83978239b5effef
+          type: localOciBlob
+        name: config
+        relation: local
+        type: yaml
+        version: 0.0.1-alpha
+      sources:
+      - access:
+          commit: 157f9933b29f107e9030b60411ee7f0e9a3bea7d
+          ref: refs/heads/main
```

Comment:
on main, RBACs are still in `*` import. This cannot be accepted for module submission.

Additionally, your controller requests `*` on get of all resources in the cluster, this is invalid.

Reply:
> on main, RBACs are still in `*` import. This cannot be accepted for module submission. Specifically, your controller requests `*` on get of all resources in the cluster, this is invalid.

We are limited with keda requirements:
- https://github.com/kyma-project/keda-manager/blob/main/module-chart/templates/10-keda-clusterrole.yaml#L37

Reply:
> On top of that, your controller has no finalizer defined in https://github.com/kyma-project/keda-manager/blob/8622f7f4eab5b64994a89eec5edd0f3e1082f8d6/controllers/keda_controller.go#L75 and will thus break on deletion of your resource by not waiting and reporting state correctly.

thx, updated in [this PR](https://github.com/kyma-project/keda-manager/commit/a0e86481d91cb126972e2cba82361c11d6e4d2db#diff-14f943bad946b1bf34063f0b9321468d99706ebc2bafec6eb129fb51ba470375R78)